js/NavMainComponent.js

```diff
@@ -2,104 +2,278 @@
 
 var $ = require('jquery');
 
-function NavMainComponent(html) {
+function NavMainComponent (html, window) {
     this.$html = html;
+    this.$window = $(window);
```

Comment:
Is `this.$window` ever being used by this component? I can't find a reference to it. If not, can we remove it from here and in `main.js`

Reply:
It is used on line 174 to detect windows height, which in turn is used for the adjustNavItems function calculations.